cypress/e2e/tests/pages/extensions/kubewarden.spec.ts

```diff
@@ -8,25 +8,30 @@ const extensionName = 'kubewarden';
 
 describe('Kubewarden Extension', { tags: ['@extensions', '@adminUser'] }, () => {
   before(() => {
+    // needed to swallow unreproducible error while running tests locally
+    // more info in https://github.com/rancher/dashboard/pull/11036
+    cy.once('uncaught:exception', () => false);
```

Comment:
can we print out the e.message and use `catchTargetPageException` to catch the specific case that's the issue?